content-src/components/DiscoveryStreamComponents/Navigation/_Navigation.scss

```diff
@@ -0,0 +1,44 @@
+.ds-navigation {
+
+  &.ds-navigation-centered {
+    text-align: center;
+  }
+
+  &.ds-navigation-right-aligned {
+    text-align: right;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  ul li {
+    display: inline-block;
+
+    &::after {
+      content: '•';
+      padding: 8px;
+    }
+
+    &:last-child::after {
+      content: none;
+    }
+
+    a {
+      color: $blue-60;
```

Comment:
The designs have hover and active styling that don't seem to be here. Also, the rest of the page had been using another level of indirection to avoid copying this `$blue-60` to each link-ish thing and instead referencing some "link-primary-color". I guess maybe we aren't doing that now but definitely helps for supporting multiple themes.